Logstash tar installation instructions (#5129)
opensearch-project/documentation-website: _tools/logstash/index.md

## Install Logstash

> Installing Logstash involves two main steps. The first is installing the Logstash itself. The second is installing the OpenSearch Logstash plugin.

Reviewer: ```suggestion
To install Logstash on OpenSearch, first install Logstash on your cluster, then the OpenSearch Logstash plugin, as described in the following steps.
```

### Tarball

Make sure you have [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) version 8 or 11 installed.

1. Download the Logstash tarball from [Logstash downloads](https://www.elastic.co/downloads/logstash).

> 2. Navigate to the downloaded folder in the terminal and extract the files (Note that the version and platform may vary from the following command):

Reviewer: ```suggestion
2. Navigate to the downloaded folder in the terminal and extract the files. Make sure that your version of Logstash and platform matches the one downloaded:
```

     ```bash
     tar -zxvf logstash-8.8.2-linux-x86_64.tar.gz
     ```

3. Navigate to the `logstash-8.8.2` directory.

> 4. Install the plugin

Reviewer: ```suggestion
4. Use the following command to install the plugin:
```
